addon/tests/helpers/with-feature.js

```diff
@@ -1,8 +1,8 @@
 import Ember from 'ember';
+import features from 'ember-feature-flags/features';
 
 export function withFeature( featureName ){
-  window.Features = window.Features || {};
-  window.Features[featureName] = true;
+  features.set(featureName, true);
```

Comment:
With the initializer auto-loading from the the config, this is actually incorrect because features are not bound, so I could turn a feature on for a test but if it was off at the first render it would not reflect in the DOM correctly.